projects/plugins/jetpack/extensions/shared/register-jetpack-block.js

```diff
@@ -16,28 +18,32 @@ import { addFilter } from '@wordpress/hooks';
  * @returns {object|boolean} Either false if the block is not available, or the results of `registerBlockType`
  */
 export default function registerJetpackBlock( name, settings, childBlocks = [], prefix = true ) {
```

Comment:
In `block.json`, `name` must be prefixed with `jetpack/`. These changes account for that.

Reply:
Maybe we could extend the function's docblock to explain what `name` can be / should be?